samples/connectivity/callnotification/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+plugins {
+    id("com.example.platform.sample")
+}
+
+android {
+    namespace = "com.example.platform.connectivity.callnotification"
+}
+
+dependencies {
+    implementation("androidx.core:core-ktx:1.10.1")
```

Comment:
This lib is already part of the base libs in gradle/libs.versions.toml

If you need a higher version you can increase it there